src/ssr-carousel.vue

```diff
@@ -13,6 +13,8 @@
 		v-on=`disabled ? {} : {
 			mousedown: onPointerDown,
 			touchstart: onPointerDown,
+			mouseenter: onEnter,
+			mouseleave: onLeave
```

Comment:
Like I mentioned above, I think we should only add these listeners if the user is using autoplay.  I think a clean way to do this would be to move all of this to a computed property.  Like:

```
v-on='maskListeners'
```

```coffee
computed:

  watchesHover: -> @autoplayDelay > 0

  maskListeners: ->
    return {} if @disabled
    {
      mousedown: @onPointerDown
			touchstart: @onPointerDown
			...(unless @watchesHover then {} else {
			  mouseenter: @onEnter
			  mouseleave: @onLeave
			})
    }